tests/global_config.py

```diff
@@ -1,12 +1,14 @@
 global config  # type:ignore[unused-ignore]
 
-distribution: str = "downstream"
-applications_namespace: str = "redhat-ods-applications"  # overwritten in conftest.py if distribution is upstream
 dsc_name: str = "default-dsc"
 must_gather_base_dir: str = "must-gather-base-dir"
 dsci_name: str = "default-dsci"
 dependent_operators: str = "servicemeshoperator,authorino-operator,serverless-operator"
 use_unprivileged_client: bool = True
+# overwrite the followings in conftest.py if distribution is upstream
```

Comment:
I would specify `tests/conftest.py`, maybe even point to the fixture name (`updated_global_config`)

Reply:
Going to keep it for now. Since technically we are invoking updated_global_config in conftest.py - which is causing the override.

Reply:
Yes, and I mean that this file should point to that fixture, since if you look at this you don't really know where the override is happening